packages/algorand-js/src/errors/errors-list.ts

```diff
@@ -133,6 +133,18 @@ by an index that does not exist.`
     message: "Pragma version Error - Expected: version, got: %got%, Line: %line%",
     title: PARSE_ERROR,
     description: ``
+  },
+  LABEL_NOT_FOUND: {
+    number: 19,
+    message: "Label not found",
+    title: "Label %label% not found",
+    description: `Label %label% not found`
+  },
+  INVALID_STACK_ELEM: {
+    number: 20,
+    message: "Invalid top of stack or length of stack > 1",
+    title: "Invalid top of stack or length of stack > 1",
+    description: `Either Length of stack is >= 1 or top of stack is not uint64`
```

Comment:
Please make sure whether length of stack is `>` or `>=` 1.

Reply:
yes